.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 yarn.lock
+.vscode
```

Comment:
This should be in your own global gitignore.

Reply:
Fair enough